ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -286,16 +533,87 @@ successful f = C.predicate $ \ev -> case eventMockResp ev of
 -- Tagging works on symbolic events, so that we can tag without doing real IO.
 -- TODO(kde) This needs extending as part of #251
 tag :: [Event Symbolic] -> [Tag]
-tag = C.classify
-    [ tagGetBinaryBlobJust
-    ]
+tag [] = [TagEmpty]
+tag ls = C.classify
+    [ tagGetBinaryBlobNothing
```

Comment:
Let's take a look at what kinds of examples QC generates for these tags (see my blog post). We don't have to ensure that they are minimal at this point (that's a lot of work), but is a useful sanity check. Maybe just copy and paste them into the PR description.